scripts/sonic-kdump-config

```diff
@@ -823,14 +823,18 @@ def main():
         help='Show kdump config in json format')
 
     # Maximum number of kernel core dumps
-    parser.add_argument('--num_dumps', nargs='?', type=int, action='store', default=False,
+    parser.add_argument('--num_dumps', action='store_true', default=False,
```

Comment:
This should be an integer I believe as number of dumps to be save is an integer.